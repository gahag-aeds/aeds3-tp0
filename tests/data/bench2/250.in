250 125
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 
Add 151 233
Sum 117 199
Sub 118 200
Min 17 99
Add 107 189
Max 165 247
Add 58 140
Sum 137 219
Add 12 94
Min 91 173
Sub 17 99
Max 162 244
Add 129 211
Sum 74 156
Sub 125 207
Sum 68 150
Sub 92 174
Sum 150 232
Add 35 117
Max 62 144
Add 98 180
Max 83 165
Sub 154 236
Sum 90 172
Sub 122 204
Min 84 166
Add 128 210
Max 140 222
Sub 39 121
Min 9 91
Sub 47 129
Sum 102 184
Sub 126 208
Min 77 159
Add 32 114
Max 65 147
Sub 154 236
Min 89 171
Sub 35 117
Min 166 248
Sub 93 175
Min 131 213
Add 160 242
Sum 54 136
Sub 37 119
Sum 30 112
Add 121 203
Min 41 123
Add 12 94
Min 68 150
Sub 102 184
Sum 110 192
Add 150 232
Sum 88 170
Sub 112 194
Max 105 187
Add 84 166
Min 73 155
Add 157 239
Max 36 118
Add 81 163
Sum 36 118
Sub 137 219
Max 119 201
Sub 25 107
Max 1 83
Sub 17 99
Min 12 94
Add 3 85
Sum 131 213
Sub 10 92
Sum 95 177
Add 94 176
Sum 82 164
Add 61 143
Sum 43 125
Add 111 193
Sum 95 177
Sub 83 165
Min 35 117
Sub 162 244
Min 17 99
Sub 57 139
Max 145 227
Add 17 99
Min 2 84
Add 162 244
Min 100 182
Sub 154 236
Max 64 146
Add 135 217
Sum 147 229
Add 100 182
Min 18 100
Sub 99 181
Sum 37 119
Sub 98 180
Max 115 197
Add 48 130
Min 13 95
Add 158 240
Min 137 219
Add 107 189
Min 84 166
Add 131 213
Min 81 163
Sub 126 208
Sum 154 236
Add 8 90
Max 41 123
Sub 22 104
Max 2 84
Add 137 219
Min 78 160
Sub 59 141
Sum 153 235
Sub 159 241
Max 133 215
Add 165 247
Sum 145 227
Add 30 112
Min 46 128
Add 38 120
Sum 42 124
Sub 63 145
