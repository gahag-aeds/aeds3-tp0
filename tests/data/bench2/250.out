4316
0
100
4325
2
100
4029
3960
4435
101
95
4113
0
99
0
3913
0
100
-1
-2
-2
3900
3906
-1
0
3853
3795
97
1
99
3935
97
100
-1
4277
3961
3821
3974
4159
-1
-1
98
0
0
98
4388
-1
3879
98
0
1
0
0
4543
100
101
0
4620
100
4544
0
4073
